src/app/forgot-password/forgotPassword.html

```diff
@@ -4,30 +4,36 @@
       <div class="panel-heading">
         <h3 class="panel-title"><strong>Forgot password</strong></h3>
         <div>
-          <a [routerLink]=" ['/Login']" id="forgot-password-login-link">Login?</a>
+          <a [routerLink]=" ['/login']" id="forgot-password-login-link">Login?</a>
         </div>
       </div>
 
       <div class="panel-body">
         <form role="form" novalidate id="forgot-password-form"
-              [ngFormModel]="form" (submit)="submit($event)" *ngIf="!validSubmit">
+              [formGroup]="form" (submit)="submit($event)" *ngIf="!validSubmit">
 
           <div class="form-group" [ngClass]="{'has-error': ( submitted && !form.controls.email.valid )}">
 
             <div class="input-group">
               <span class="input-group-addon"><i class="glyphicon glyphicon-envelope"></i></span>
-              <input id="forgot-password-email" type="text" class="form-control" name="email" value=""
-                     placeholder="Your email"
-                     required
-                     ngControl="email">
+              <input
+                id="forgot-password-email"
+                type="text"
+                class="form-control"
+                name="email"
+                value=""
+                placeholder="Your email"
+                formControlName="email"
+                required
+              >
             </div>
 
             <div class="error-group" *ngIf="( submitted && !form.controls.email.valid )">
               <span class="help-block" *ngIf="form.controls.email.hasError('required')">
                 Your email is required
               </span>
               <span class="help-block"
-                    *ngIf="!form.controls.email.hasError('required') && form.controls.email.hasError('invalidEmail')">
+                    *ngIf="!form.controls.email.hasError('required') && form.controls.email.hasError('validEmail')">
```

Comment:
doesn't make sense to me that "validEmail" is an error. See related comment in 
src/app/email-validator/email.validator.ts